tools/pythonpkg/src/python_conversion.cpp

```diff
@@ -274,8 +274,12 @@ Value TransformPythonValue(py::handle ele, const LogicalType &target_type, bool
 		return Value::UUID(string_val);
 	}
 	case PythonObjectType::Datetime: {
-		auto isnull_result = py::module::import("pandas").attr("isnull")(ele);
-		bool is_nat = string(py::str(isnull_result)) == "True";
+		auto &import_cache = *DuckDBPyConnection::ImportCache();
+		bool is_nat = false;
+		if (import_cache.pandas.isnull.IsLoaded()) {
```

Comment:
Should we check if `pandas` is loaded instead? Or does that not matter because we always call `LoadSubtypes`?

Reply:
Hmm no, LoadSubtypes would not be called in this case because `pandas` failed to load, but PythonImportCacheItem sets `object` to nullptr, which is what IsLoaded checks for